internal/pally/counter.go

```diff
@@ -23,7 +23,9 @@ package pally
 import (
 	"fmt"
 
+	"github.com/golang/protobuf/proto"
```

Comment:
The bigger thing is I want the whole company to get into the habit of using gogo over golang, most of the community is moving to gogo (including even google at this point, kubernetes uses it a bit now). The APIs are compatible, it should be a drop-in replacement. Types generated by either work with either. If you s/golang/gogo above, it should just work.